### ISSUE TITLE
Minimum main window height is too tall

### ISSUE THREAD

**Issue by geary** [RickStrahl/MarkdownMonster#326]
I'm running Windows 10 (latest edition) on a ThinkPad with a WQHD display (2560x1440) at 225% scaling. So the logical display height is 640 pixels.

Markdown Monster has `MinHeight="700"` in [line 23 of MainWindow.xaml](https://github.com/RickStrahl/MarkdownMonster/blob/f78f5f4f740c40508a42994f4c33577170a3ffb0/MarkdownMonster/MainWindow.xaml#L23). As a result, the status line and the bottom of the editor and preview panels are clipped off the screen when I maximize the main window. In fact, the first few times I ran MM I didn't even realize it had a status line with settings I could tweak directly.

I tried building from source with `MinHeight` reduced to 400 pixels and that fixed the problem. There doesn't seem to be any reason to require a taller height.

It may also desirable to reduce the `MinWidth` from the current 800 pixels. While some of the toolbar icons do get clipped off at a narrower width, they also get clipped at 800 pixels. And someone may want to edit in a smaller window, e.g. to look at something else on the screen while they edit.

I would be happy to send a PR, although it is a fairly trivial change and I wanted to get your thoughts on the smaller minimum sizes first. Thanks!

**Comment by RickStrahl** [comment]
I've gone back and forth on this. One reason for the width is the icons - I want to try and make sure the icons are visible. The height is adjusted to be reasonably square in that porportion. I guess I could drop the height, but man...  640px real height is <small>tiny</small> :-) I would almost tag that as unusable for MM.

**Comment by geary** [comment]
I've tried running this display at 200% scaling instead, which gives a logical height of 720 pixels, but I don't like the text sizes I get in other apps/websites at that scaling. 225% is more comfortable for me.

MM is perfectly usable on the WQHD at 225% scaling once the MinHeight is reduced. I did notice that the default editor window text was very large with this scaling and I reduced it to 14px. The text is still big enough, and this gives me close to 30 lines of text in the editor window, which is plenty for my needs.

Sure, more height would be better for many things - that's why my second monitor is a 24" 4K UHD in _portrait mode_ next to the laptop display, which is great for things like viewing PDF files. In fact, it was when I happened to move the MM window over to that display that I first noticed it even _had_ a status line!

In any case, in this era of 16:9 displays, the "reasonably square" ship sailed long ago. I have to work with the displays I have today, not the 4:3 displays I used to use.

It's also worth considering the other use case I mentioned, where I want to refer to something else on the screen while editing (and let's say I am away from the office and just have the laptop with me). So I may want to reduce the MM window size even further for a short time. If the toolbar icons get clipped off, that's OK. (But again that would be a question of the width - it's only the minimum height that is actually causing me a problem.)
